Dockerfile

```diff
@@ -1,17 +1,25 @@
 # Build the manager binary
 FROM golang:1.10.3 as builder
 
+# Install Dep
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
 # Copy in the go src
 WORKDIR /go/src/github.com/pusher/wave
+COPY Gopkg.lock Gopkg.lock
+COPY Gopkg.toml Gopkg.toml
 COPY pkg/    pkg/
 COPY cmd/    cmd/
-COPY vendor/ vendor/
+
+# Fetch dependencies
+RUN dep ensure --vendor-only
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager github.com/pusher/wave/cmd/manager
 
 # Copy the controller-manager into a thin image
-FROM ubuntu:latest
-WORKDIR /root/
+FROM alpine
```

Comment:
Should we perhaps have a specific version rather than implicitly using `latest`?

Reply:
We probably should have that in both the wercker and dockerfile definitions, I'll pin to `3.8` for now